src/Intl/Idn/Idn.php

```diff
@@ -280,10 +280,6 @@ private static function mapCodePoints($input, array $options, Info $info)
 
             switch ($data['status']) {
                 case 'disallowed':
-                    $info->errors |= self::ERROR_DISALLOWED;
```

Comment:
Can you explain why this change?

Reply:
The reasoning for this change according to the spec is "Checking for disallowed characters both before and after normalization yielded inconsistent results for unnormalized vs. normalized text".